diff --git a/.npmignore b/.npmignore
deleted file mode 100644
index 5f82b42bc0480aaaaeb95d299c25172bf9f95e7c..0000000000000000000000000000000000000000
diff --git a/src/lib/mark.js b/src/lib/mark.js
index 958e6184e17ebd2566c410c4de7349b6ae92966f..b0597df80788445897e974579d26d4b1eb2a7b88 100644
--- a/src/lib/mark.js
+++ b/src/lib/mark.js
@@ -1,4 +1,4 @@
-import DOMIterator from './domiterator';
+import DOMIterator from './domiterator.js';
 
 /**
  * Marks search terms in DOM elements
diff --git a/src/vanilla.js b/src/vanilla.js
index 3636b4335b93ff87effffdc3734088e1e7c8cf10..5d742a878692218c1b3822af6ce60fdbf4938fba 100644
--- a/src/vanilla.js
+++ b/src/vanilla.js
@@ -1,4 +1,4 @@
-import MarkJS from './lib/mark';
+import MarkJS from './lib/mark.js';
 
 export default function Mark(ctx) {
   const instance = new MarkJS(ctx);
